src/components/ui-shell/_navigation-menu.scss

```diff
@@ -104,12 +109,12 @@
   }
 
   .#{$prefix}--navigation__category-toggle:hover {
-    background-color: $ibm-colors__gray--80;
+    background-color: #333333;
   }
 
   .#{$prefix}--navigation__category-toggle:focus {
-    outline: mini-units(0.5) solid $ibm-colors__blue--60;
-    outline-offset: mini-units(-0.5);
+    outline: rem(3px) solid $ibm-colors__blue--60;
```

Comment:
this is a bit confusing now, because it was initially 4px, but I may have misinterpreted the comments made [here](https://github.com/IBM/carbon-components/pull/1529#pullrequestreview-181972702) on focus outline width